.codeclimate.yml

```diff
@@ -1,6 +1,11 @@
 engines:
   rubocop:
     enabled: true
+    exclude_fingerprints:
+    # Ignoring long method length for Issue#to_json
+    - 3d618821b1ce28599d6c54f90bb4df59
 ratings:
   paths:
   - "**.rb"
+exclude_paths:
+  - bin/bundler_audit.rb
```

Comment:
Rubocop generated an error on the shebang line. I think we usually drop the extension on executables in other engines, which is why this doesn't often need to be excluded. Might be worth doing that instead.